jbpm-wb-executor-service/jbpm-wb-executor-service-client/src/main/java/org/jbpm/workbench/es/client/editors/errorlist/ExecutionErrorListPresenter.java

```diff
@@ -353,6 +353,36 @@ public void setupActiveSearchFilters() {
             isDefaultFilters = false;
         }
         
+        final Optional<String> taskIdSearch = getSearchParameter(PerspectiveIds.SEARCH_PARAMETER_TASK_ID);
+        if (taskIdSearch.isPresent()) {
+            final String taskId = taskIdSearch.get();
+            view.addActiveFilter(constants.Task(),
+                                 taskId,
+                                 taskId,
```

Comment:
Why are we passing the same argument twice?

Reply:
The first one is the label value that is shown in the top of the view and the second one is the real value by which we are filtering. I guess that in this case it's the same value, but it could be any other cases with different values.